client/ladder/action-creators.ts

```diff
@@ -12,6 +12,13 @@ const lastFetchTimeByMatchmakingType = new Map<MatchmakingType, number>()
 const lastSearchTimeByMatchmakingType = new Map<MatchmakingType, number>()
 const lastSearchQueryByMatchmakingType = new Map<MatchmakingType, string>()
 
+const lastFetchTimeBySeasonIdAndMatchmakingType = new Map<SeasonId, Map<MatchmakingType, number>>()
```

Comment:
These could both be converted to strings so you don't need to keep a nested Map, e.g. `${seasonId}|${matchmakingType}`

Reply:
I also don't really get why the ones above here (that don't include a season ID) still exist?